### DISCUSSION TITLE
Why are there two copies of spam lists in the resource?

### DISCUSSION THREAD

**Discussion by Firecul** [ThioJoe/YT-Spammer-Purge#673] [Q&A] [answered]
Related to our talk in https://github.com/ThioJoe/YT-Spammer-Purge/discussions/666#discussioncomment-2197727 I was working through the code trying to figure out why there is a copy of all 3 spam lists in both the `assets` and `SpamPurge_Resources` folders?

**Comment by ThioJoe** [comment, marked as answer]
The one in the assets folder is basically the default that gets loaded the first time and gets put into the SpamPurge_Resources folder. Basically just in case for some reason the program can't fetch the latest spam list, it has something to use.

**Reply by Firecul** [reply]
~~If the assets folder version is the default and gets loaded into the resources folder automatically wouldn't it be easier to update by omitting the resources version as downloaded from the repo then?~~
